packages/core/src/app/shipping/hooks/useMultishippingConsignmentItems.ts

```diff
@@ -10,6 +10,15 @@ interface MultiShippingConsignmentItemsHook {
     consignmentList: MultiShippingConsignmentData[];
 }
 
+// TODO: consolidate this from /app/order/removeBundledItems
+function removeBundledItems(lineItems: LineItemMap): LineItemMap {
+    return {
+        ...lineItems,
+        physicalItems: lineItems.physicalItems.filter((item) => typeof item.parentId !== 'string'),
+        digitalItems: lineItems.digitalItems.filter((item) => typeof item.parentId !== 'string'),
```

Comment:
Can we not directly check `!item.parentId`, like https://github.com/bigcommerce/manual-order-ui/blob/main/src/Consignment/hooks/useMultiShippingConsignmentItems.tsx#L60

Reply:
We can do that too, but this feels a better check in my opinion. Plus after this I am going to consolidate this with the existing function that has the bundled item logic.

Reply:
Manual order does not filter out digital child item, perhaps we should add it asap. 🤦🏻‍♂️